test/shaping/run-tests.py

```diff
@@ -63,6 +80,12 @@ def cmd(command):
 			print ("%s %s %s %s --unicodes %s" %
 					 (hb_shape, fontfile, ' '.join(extra_options), options, unicodes))
 
+		# hack to support fonts with space on run-tests.py, after several other tries...
+		if ' ' in fontfile:
```

Comment:
Ouch!

Reply:
This hack needs to be fixed...

Actually, Codacy warns about using Subprocess without shell=true.  Is that relevant? https://app.codacy.com/project/behdad/harfbuzz/dashboard